### PR TITLE
feature 37408 - Added assignee

### PR DESCRIPTION
**Pull request by Rick-Ritense**
Added assignee first and last names.

### CODE REVIEW

File: document/src/main/resources/config/liquibase/changelog/20220923-add-assginee-columns-changelog.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="Rick Veenstra" id="1">
+        <addColumn tableName="json_schema_document">
```
Comment (Klaas-Ritense) [inline document/src/main/resources/config/liquibase/changelog/20220923-add-assginee-columns-changelog.xml:5]: The database should also have some kind of an identifier for the assignee. id/email

**Comment by ivo-ritense** [comment]
I feel like we're missing a test to see if it's actually returned in the endpoint.

**Review by Klaas-Ritense — APPROVED** [review]

File: document/src/main/resources/config/liquibase/changelog/20220923-add-assginee-columns-changelog.xml (hunk 1 above)
Comment (Klaas-Ritense) [reply document/src/main/resources/config/liquibase/changelog/20220923-add-assginee-columns-changelog.xml:6]: I've added it

File: document/src/main/java/com/ritense/document/domain/Document.java [hunk 2]
```diff
@@ -42,6 +43,15 @@ public interface Document {
     @JsonProperty
     DocumentDefinition.Id definitionId();
 
+    @JsonProperty
+    String assigneeEmail();
```
Comment (ivo-ritense) [inline document/src/main/java/com/ritense/document/domain/Document.java:47]: Even though i don't mind having the email, i think we always also want the user ID

Comment (Klaas-Ritense) [reply document/src/main/java/com/ritense/document/domain/Document.java]: change email to id

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**